tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -69,13 +69,18 @@ OPTIMIZED_KERNEL_DIR_PREFIX := tensorflow/lite/micro/kernels
 # be used.
 CO_PROCESSOR :=
 
+# Compier flags for OpenCV libraries and header files.
```

Comment:
technically you could probably get away with putting these changes in the example-specific Makefile.inc.